yaml/csr.yaml.m4

```diff
@@ -4410,7 +4414,7 @@ ifelse(eval(CLIC != 0), 1, [[[
         val_out = 0
 ]]])
 
-ifelse(eval(CLIC != 0), 1, [[[
+ifelse(eval(CLIC != 0 && READONLY != 0), 1, [[[
```

Comment:
How/when/why is READONLY set/cleared?

Reply:
> [How] is READONLY set/cleared?

`m4 .... -D READONLY=0`  (Or 1.)

> [When] is READONLY set/cleared?

Anyone who uses m4 to generate a specific YAML **must** define all variables which are used in `eval` expressions.

> [Why] is READONLY set/cleared?

If it is not defined one gets an error `[...] bad expression in eval`.
We **clear** it in verif when generating the CSR access test, because the test generator doesn't seem to support all the variation found in RISC-V CSR behavior.

Reply:
It is not new.